src/servers/graphql-server.ts

```diff
@@ -487,17 +490,41 @@ export const startApolloServer = async ({
             execute,
             subscribe,
             onConnect: async (ctx) => {
+              const kratos_cookie = ("; " + ctx.extra.request.headers.cookie)
+                .split(`; ory_kratos_session=`)
+                .pop()
+                ?.split(";")[0]
```

Comment:
Yes, if we want a generic cookie parser maybe it would be good to add a well used [dependency](https://yarnpkg.com/package/@types/cookie) for it @nicolasburtey ?
The code you highlighted was simply the easiest one-liner I could find online.

Reply:
`cookie` is already a dependency: https://github.com/GaloyMoney/galoy/blob/main/package.json#L61

Reply:
Updated to use that library in order to parse the cookie.